README.md

```diff
@@ -24,42 +24,42 @@ This application allows you to integrate OpsGenie to your Mattermost instance, l
 
 ## Installation
 
-This plugin requires that your Mattermost workspace has the ``/apps install`` command enabled.
+This plugin requires that your Mattermost workspace has the `/apps install` command enabled.
 
-To install, as a super admin user role, execute command /apps install http OPSGENIE_API_URL in any channel. /genie command should be available after the configuration has been successfully installed. OPSGENIE_API_URL should be replaced with the URL where the OpsGenie API instance is running. Example: /apps install http https://mattermost-opsgenie-dev.ancient.mx/manifest.json
+To install, as a super admin user role, execute command `/apps install` http OPSGENIE_API_URL in any channel. The `/genie` command should be available after the configuration has been successfully installed. The `OPSGENIE_API_URL` should be replaced with the URL where the OpsGenie API instance is running. Example: `/apps install http https://mattermost-opsgenie-dev.ancient.mx/manifest.json`.
 
 ## Configuration
 
-Configuration Step 1: Firstly, you need to install the app in your current Mattermost instance (refer to Installation), the /genie command should be available.
-Step 2: Configure OpsGenie Api-Key As a super admin role user, execute /genie configure command, which will open a modal were the API key will be asked (https://example.app.opsgenie.com/settings/integration/edit/API/[id]). To be able to obtain the OpsGenie Api-Key first you have to add an api integration to your OpsGenie account, you can create the integration going to the settings/integrations tab and clicking the “Add Integration” button. Once inside the “Add Integration” menu, select the “API” option. To finish the api integration setup you just need to click the “Save Integration” button
+1. Firstly, you need to install the app in your current Mattermost instance (refer to Installation), the `/genie` command should be available.
+2. Configure OpsGenie Api-Key As a super admin role user, execute `/genie` configure command, which will open a modal where the API key will be asked (https://example.app.opsgenie.com/settings/integration/edit/API/[id]). To be able to obtain the OpsGenie Api-Key first you have to add an API integration to your OpsGenie account, you can create the integration going to the **Settings/Integrations** tab and clicking **Add Integration**. Once inside the **Add Integration** menu, select the **API** option. To finish the API integration setup select **Save Integration**.
 
 # Admin guide
 
 ## Slash commands
 
-/pd configure: This command will enable all the other commands; it asks the administrator for an API key (which will be used to execute calls to OpsGenie’s API)
+`/pd configure`: This command will enable all the other commands; it asks the administrator for an API key (which will be used to execute calls to OpsGenie’s API)
 
 # End user guide
 
 ## Getting started
 
 ## Using /genie commands
```

Comment:
@development-ancient, I changed these to "genie" but am not sure if they're all correct/meant for this doc set. Please confirm, thank you!